
From: Steffen Klassert <klassert@mathematik.tu-chemnitz.de>

Patch changes the two remaining direct accessing of dev->priv to netdev_priv.

Signed-off-by: Steffen Klassert <klassert@mathematik.tu-chemnitz.de>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/net/3c59x.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff -puN drivers/net/3c59x.c~3c59x-use-netdev_priv drivers/net/3c59x.c
--- 25/drivers/net/3c59x.c~3c59x-use-netdev_priv	2004-09-30 22:47:26.058574848 -0700
+++ 25-akpm/drivers/net/3c59x.c	2004-09-30 22:47:26.074572416 -0700
@@ -933,7 +933,7 @@ static int vortex_cards_found;
 #ifdef CONFIG_NET_POLL_CONTROLLER
 static void poll_vortex(struct net_device *dev)
 {
-	struct vortex_private *vp = (struct vortex_private *)dev->priv;
+	struct vortex_private *vp = netdev_priv(dev);
 	unsigned long flags;
 	local_save_flags(flags);
 	local_irq_disable();
@@ -2981,7 +2981,7 @@ static void set_rx_mode(struct net_devic
 
 static void set_8021q_mode(struct net_device *dev, int enable)
 {
-	struct vortex_private *vp = (struct vortex_private *)dev->priv;
+	struct vortex_private *vp = netdev_priv(dev);
 	long ioaddr = dev->base_addr;
 	int old_window = inw(ioaddr + EL3_CMD);
 	int mac_ctrl;
_
